website/static/js/contribManager.js

```diff
@@ -113,6 +116,37 @@ var ContributorModel = function(contributor, currentUserCanEdit, pageOwner, isRe
             id:self.id});
     };
 
+    self.addParentAdmin = function () {
+        // Immediately adds parent admin to the component with permissions=read and visible=True
+        var self = this;
+        $osf.block();
+        self.nodeId = window.contextVars.node.id;
```

Comment:
I haven't looked at Javascript in *quite* sometime, but why
```
self.nodeId = ...
self.nodeApiUrl = ...
``` 
instead of 
``` 
var nodeId = ...
var nodeApiUrl = ...
```
?

If these variables need to be defined on the `ContributorModel`, can they be placed outside of this function?

Reply:
thank you, they don't need to be defined on the ContributorModel, I don't really need to define them at all, since they are only used once.  I'll move this to where it's used.